DigitalLearningSolutions.Web/Scripts/searchSortFilterAndPaginate/searchSortFilterAndPaginate.ts

```diff
@@ -67,11 +81,15 @@ export class SearchSortFilterAndPaginate {
   }
 
   private searchSortAndPaginate(searchableData: ISearchableData): void {
-    const searchedElements = search(searchableData.searchableElements);
+    const searchedElements = this.searchEnabled
+      ? search(searchableData.searchableElements)
+      : searchableData.searchableElements;
     const filteredElements = this.filterEnabled
       ? filterSearchableElements(searchedElements, searchableData.possibleFilters)
       : searchedElements;
-    const sortedElements = sortSearchableElements(filteredElements);
+    const sortedElements = this.sortEnabled
+      ? sortSearchableElements(filteredElements)
+      : filteredElements;

```

Comment:
Have you checked the behaviour of the other pages, this change will impact the ordering of results on pages the 'don't have' sort (ones with hiddens where we just default to the title of the expanders)